src/components/v5/common/ActionSidebar/partials/ActionSidebarContent/ActionSidebarContent.tsx

```diff
@@ -30,36 +31,33 @@ import useHasActionPermissions from '../../hooks/permissions/useHasActionPermiss
 import useHasNoDecisionMethods from '../../hooks/permissions/useHasNoDecisionMethods.ts';
 import useActionFormProps from '../../hooks/useActionFormProps.ts';
 import useSidebarActionForm from '../../hooks/useSidebarActionForm.ts';
+import { type ActionFormBaseProps } from '../../types.ts';
 import ActionButtons from '../ActionButtons.tsx';
 import ActionSidebarDescription from '../ActionSidebarDescription/ActionSidebarDescription.tsx';
-import Motions from '../Motions/index.ts';
 import RemoveDraftModal from '../RemoveDraftModal/RemoveDraftModal.tsx';
 
 import { useGetFormActionErrors } from './hooks.ts';
 import NoPermissionsError from './partials/NoPermissionsError.tsx';
 import NoReputationError from './partials/NoReputationError.tsx';
-import PermissionSidebar from './partials/PermissionSidebar.tsx';
 import { SidebarBanner } from './partials/SidebarBanner.tsx';
-import {
-  type ActionSidebarContentProps,
-  type ActionSidebarFormContentProps,
-} from './types.ts';
+import { type ActionSidebarContentProps } from './types.ts';
 
 const displayName = 'v5.common.ActionsContent.partials.ActionSidebarContent';
 
-const ActionSidebarFormContent: FC<ActionSidebarFormContentProps> = ({
+const ActionSidebarFormContent: FC<ActionFormBaseProps> = ({
```

Comment:
I'm not really a big fan of having two React components made inside one file, especially ones this size, so I'd prefer that they were split into two files. I do see the reasoning behind doing it like this since they're so tightly coupled, but I do think it'd be cleaner to separate them. Again, happy to hear opposing opinions though!

Reply:
I agree. But it's not something that I introduced in this PR :P